_plugins/jekyll_copy_uswds.rb

```diff
@@ -0,0 +1,12 @@
+module Jekyll
```

Comment:
Do you have to declare the plugin in the config?

Reply:
nope, it will load anything in that folder.
https://jekyllrb.com/docs/plugins/installation/

Reply:
Ah yes. The cheesecake factory of plugin options.